src/content/migrate/4.md

```diff
@@ -37,7 +44,9 @@ module.exports = {
 }
 ```
 
-Alternatively you can pass it via CLI: `--mode production`/`--mode development`
+Alternatively you can pass it via CLI: `--mode production`/ `--mode development`
```

Comment:
this is the first thing it says on the mode page we are linking in the beggining of this section, for the sake of duplication prevention, i suggest to remove this line

Reply:
corrected.